README.md

```diff
@@ -174,10 +165,13 @@ This allows to parallelize builds across multiple machines.
 
 ## Github api token
 
-In case your IP exceeds the rate limit, github will return an 403 error message.
-To increase your limit first create a [personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
-Then use either the `--token` parameter of the `pr` subcommand or
-set the `GITHUB_OAUTH_TOKEN` environment variable.
+The `pr --comment` option requires a Github API token, and even for read-only
+calls github returns 403 error messages if your IP hits the rate limit for
+unauthenticated calls.
+
+To use a token, first create a [personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
+Then use either the `--token` parameter of the `pr` subcommand or set the
```

Comment:
Please add a check in `pr_command` to verify if token was set if the comment option was passed.

Reply:
If it is not set the error message should contain a link to this section of the README.